<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: rgba(30, 30, 30, 0.9); /* Dark background with 80% opacity */
	
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="HeaderFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="login_label">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>LOGIN.png</pixmap>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:700; color:#636363;&quot;&gt;LOGIN TO &lt;/span&gt;&lt;span style=&quot; font-size:16pt; color:#3a3a3a;&quot;&gt;YOUR ACCOUNT&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	color: rgb(255, 255, 255);
    border-style: none;
    border-bottom-style: solid;
    border-bottom-width: 1px;
    border-color: rgb(61, 61, 61); /* Adjust the color as needed */
    background-color: transparent;
	font-size: 14px;
    font-family: &quot;Times New Roman&quot;, serif; /* Replace with your desired font */
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Username_lineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwrod_lineEdit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="email_lineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>user.svg</pixmap>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>lock.svg</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>mail.svg</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="BottomFrame">
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: 2px solid #4CAF50; /* Green border */
    color: #ffffff; /* White text color */
    background-color: #4CAF50; /* Green background */
    padding: 10px 20px; /* Padding around text */
    font-size: 16px; /* Font size */
    font-weight: bold; /* Bold text */
    border-radius: 5px; /* Rounded corners */
}

QPushButton:hover {
    background-color: #45a049; /* Darker green on hover */
    border-color: #45a049; /* Darker green border on hover */
}

QPushButton:pressed {
    background-color: #3e8e41; /* Dark green when pressed */
    border-color: #3e8e41; /* Dark green border when pressed */
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>18</number>
      </property>
      <item>
       <widget class="QPushButton" name="login_pushButton">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignBottom">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#aaaaaa;&quot;&gt;Â© 2024 Ndeze Bonheur.&lt;/span&gt;&lt;span style=&quot; color:#9e9e9e;&quot;&gt; All rights reserved.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>login_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>email_lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>login_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>passwrod_lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>login_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Username_lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
